docs/usage.md

```diff
@@ -414,6 +437,10 @@ Set to receive plain-text e-mails instead of HTML formatted.
 
 Set to disable colourful command line output and live life in monochrome.
 
+### `--multiqc`
+
+Path to the MultiQC zipped folder containing the MultiQC plots generated by the RNAseq pipeline, the MultiQC html report, and the MultiQC raw data.
+
```

Comment:
This is not a core param, so you could move it to the section above with the rest of `rnadeseq` specific params.

Reply:
At the moment it is optional, right?

Reply:
Yes, I moved it as you suggested and added that it is an optional param (as well as the citest param, even though that is just for development)